packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -61,5 +61,9 @@
   "pci_database_add_spec_nodes_range_single": "{{min}} nœud inclus, jusqu'à {{max}} nœuds",
   "pci_database_add_show_command": "Voir l'équivalent API",
   "pci_database_add_spec_name_popover": "Le nom de votre cluster est généré automatiquement. Vous pourrez le modifier par la suite.",
-  "pci_database_add_spec_logo_alt": "Logo du moteur"
+  "pci_database_add_spec_logo_alt": "Logo du moteur",
+  "pci_database_add_network_mode_helper": "Pour en savoir plus sur la configuration d'un réseau privé, consultez le guide : {{ link }}",
+  "pci_database_add_network_mode_helper_link": "vRack Public Cloud",
```

Comment:
is "vRack Public Cloud" needs translation ? If not we can move this to constant.

Reply:
Hello @sachinrameshn, we can let it into the content.

Reply:
Could you please elaborate @mohammed-zahaf?
Translation file isn't meant to store static content.

If we don't expect any translation regarding actual content, then we should avoid relying on translation files.

Reply:
I have this reflection because there are some technologies hasn't same name in others countries.
so, it's better to let content team to keep same content or update it and if it change in future it will be more easier to update.